workflow/commit-messages.md

```diff
@@ -0,0 +1,34 @@
+# Commit Guidelines
```

Comment:
Since we Jekyllised this repo, now documents should include the title and a description in YAML frontmatter:

``` markdown
---
title: Commit guidelines
description: {Come up with a nice description}
---

... the rest of the document
```

Also, we use sentence case rather than title case for titles, so "guidelines" should have a small "g". (Somewhat obscurely mentioned in [this practice](https://github.com/canonical-webteam/practices/blob/master/content/copy-reviews.md#checklist)).